src/weather_model_graphs/create/archetype.py

```diff
@@ -61,26 +77,30 @@ def create_graphcast_graph(
 
     Parameters
     ----------
-    xy_grid: np.ndarray
-        2D array of grid point positions.
-    grid_refinement_factor: float
-        Refinement factor between grid points and bottom level of mesh hierarchy
+    coords: np.ndarray
+        2D array of grid point positions, either in-projection Cartesian coordinates or lat-lons
+    mesh_node_distance: float
+        Distance (in x- and y-direction) between created mesh nodes,
+        in coordinate system of coords
     level_refinement_factor: int
         Refinement factor between grid points and bottom level of mesh hierarchy
         NOTE: Must be an odd integer >1 to create proper multiscale graph
     max_num_levels: int
         The number of levels of longer-range connections in the mesh graph.
+    projection: cartopy.crs.CRS or None
```

Comment:
I think this docstring line needs updating to reflect that `coords_crs` and `graph_crs` are now provided separately.